src/prerequisites/forms.py

```diff
@@ -49,13 +49,43 @@ class PrereqFormInline(FutureModelForm):
 
     prereq_object = ContentObjectChoiceField(
         queryset=QuerySetSequence(*[klass.objects.all() for klass in hardcoded_prereq_model_choice()]),
-        widget=CustomContentObjectSelect2Widget(),
+        widget=ContentObjectSelect2Widget(
```

Comment:
Can we DRY out this code by returning these search fields in a method that generates the dictionary based off of each model's "search field" method (previously called something like `dal_autocomplete_field` which existed in the IsAPrereqMixin class and was overridden by models that didn't have a `name` field.

Reply:
see https://github.com/bytedeck/bytedeck/pull/1284